File: Java/Dependency Inversion Principle/Bildirim.java
public class Bildirim implements  İletisim {

    public void bildirimGonder(String ip, String mesaj) {

        System.out.println(ip + " adresine " + mesaj + " mesajı gönderildi");
    }

    @Override
    public void bilgiGonder(Kullanici kullanici, String mesaj) {
        
        bildirimGonder(kullanici.getIp(), mesaj);
    }
}

File: Python/Dependency Inversion Principle/Bildirim.py
from İletisim import İletisim
from Kullanici import Kullanici

class Bildirim(İletisim):
    
    def BildirimGönder(self,ip,mesaj):
        print(f"{ip} adresine {mesaj} mesajı gönderildi")
        
    def Bilgilendir(self,kullanici,mesaj):
        self.BildirimGönder(kullanici.getİp(),mesaj)
        
    
